client/src/components/App/App.jsx

```diff
@@ -1,158 +1,27 @@
-                    >
-                        Material UI Theming
-                    </Link>
-                    {' works as it will help you style your application in a consistant and adaptable manner.'}
-                </span>
-            )
-        },
-        {
-            key: 'appearance',
-            icon: BrightnessIcon,
-            html: (
-                <span>
-                    {'Users might have a preference for a light or dark appearance. This preference may be specified by a system-wide setting exposed by the Operating System or by a setting controlled by the User Agent. This application uses the \'prefers-color-scheme\' media query to automatically switch to the '}
-                    <Link
-                        color="secondary"
-                        className={styles.link}
-                        href="https://material-ui.com/customization/palette/#dark-mode"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                    >
-                        Material UI Dark Mode
-                    </Link>
-                    {' when the user has specified a preference for a dark appearance.'}
-                </span>
-            )
-        }
-    ];
-
     return (
-        <div className={styles.root}>
-            <AppBar component="header" position="static">
-                <Toolbar>
-                    <Typography variant="h6">
-                        React | Material UI
-                    </Typography>
-                </Toolbar>
-            </AppBar>
-            <Container component="main" maxWidth="lg" className={styles.main}>
-                <div className={styles.content}>
-                    <img className={styles.image} src={sampleImage} alt="Sample" />
-                    <Grid container spacing={3}>
-                        {
-                            sections.map(({ icon: Icon, html, key }) => (
-                                <Grid
-                                    key={key}
-                                    item
-                                    xs={12}
-                                    sm={6}
-                                    md={12}
-                                    component={Typography}
-                                    variant="body1"
-                                    className={styles.paragraph}
-                                >
-                                    <Icon className={styles.icon} color="primary" />
-                                    {html}
-                                </Grid>
-                            ))
-                        }
-                    </Grid>
-                </div>
-            </Container>
-        </div>
+        <Router>
+            <div className={styles.root}>
+                <Header />
+                <Switch>
+                    <Route exact path="/" component={HomePage} />
+                    <Route component={Routes} />
```

Comment:
is there any need for nested routing here? or could the homepage path be included in `Routes.jsx`?